workshop.wwwapi/Endpoints/SurgeryEndpoint.cs

```diff
@@ -11,23 +12,108 @@ public static void ConfigurePatientEndpoint(this WebApplication app)
             var surgeryGroup = app.MapGroup("surgery");
 
             surgeryGroup.MapGet("/patients", GetPatients);
+            surgeryGroup.MapGet("/patientById/{id}", GetPatientById);
             surgeryGroup.MapGet("/doctors", GetDoctors);
+            surgeryGroup.MapGet("/doctorsById/{id}", GetDoctorsById);
             surgeryGroup.MapGet("/appointmentsbydoctor/{id}", GetAppointmentsByDoctor);
+            surgeryGroup.MapGet("/appointmentsbypatient/{id}", GetAppointmentsByPatient);
+            surgeryGroup.MapGet("/appointmentsbycompositeid/{doctorid} {patientid} {booking}", GetAppointmentsByCompositeId);
+            surgeryGroup.MapPost("/createPatient", CreatePatient);
+            surgeryGroup.MapPost("/createDoctor", CreateDoctor);
+            surgeryGroup.MapPost("/createAppointment", CreateAppointment);
+
+        }
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public static async Task<IResult> CreatePatient(IRepository repository, string FullName)
```

Comment:
better wrap FullName in a payload object so that you can ensure the full name must come as part of the JSON body of the request